pkg/job/job.go

```diff
@@ -63,12 +64,25 @@ type Config struct {
 	ReleaseImage string `envconfig:"OPENSHIFT_INSTALL_RELEASE_IMAGE" default:"quay.io/openshift-release-dev/ocp-release@sha256:eab93b4591699a5a4ff50ad3517892653f04fb840127895bb3609b3cc68f98f3"`
 }
 
-func New(log logrus.FieldLogger, kube client.Client, cfg Config) *kubeJob {
+func (c *Config) FixEndpointURL() error {
```

Comment:
why do you need it? why not set the url to https from the start?

Reply:
I thought that internally when we're using scality we needed http.

If not then, yeah, I can get rid of the whole env var.

Reply:
I think that you will still needed (you can probably check)
but why not to do it in main?

Reply:
> but why not to do it in main?

Instead of the `job.New`? I could do that if you prefer. But I figured if we had a single entrypoint to create a job then we could ensure it was fixed for all current callers and future callers if we do it there.